crates/common/src/result.rs

```diff
@@ -135,7 +135,13 @@ pub struct WasmError {
 macro_rules! wasm_error {
     ($e:expr) => {
         $crate::WasmError {
-            file: file!().to_string(),
+            // On Windows the `file!()` macro returns a path with inconsistent formatting:
+            // from the workspace to the package root it uses backwards-slashes,
+            // then within the package it uses forwards-slashes.
+            // i.e. "test-crates\\wasm_core\\src/wasm.rs"
+            //
+            // To remedy this we normalize the formatting here.
+            file: file!().replace('\\', "/").to_string(),
```

Comment:
This is the only issue I found noting this: https://github.com/rust-lang/libs-team/issues/503

Alternatively we could just keep the inconsistent path formatting. Or I could use a real "path canonicalization" lib instead of a replace().